invokeai/frontend/web/src/features/stylePresets/hooks/usePresetModifiedPrompts.ts

```diff
@@ -0,0 +1,27 @@
+import { useAppSelector } from 'app/store/storeHooks';
+
+export const PRESET_PLACEHOLDER = `{prompt}`;
```

Comment:
Went with `[prompt]` per @hipsterusername

Reply:
That could cause issues if we do imports from other apps, where `[...]` means "downweight this part of the prompt". Maybe `%prompt%`? That's not used in any existing prompt syntax AFAIK.

Reply:
Since we will be replacing client-side, I feel like that would be a major edge case that someone _wants_ to downweight the string "prompt" in their prompt